src/app/tasks/task-viewer/task-viewer.less

```diff
@@ -71,3 +71,11 @@
     .dropdown-selector;
   }
 }
+
+
+.task-viewer-tabs {
+  li {
+    text-align:center!important;
```

Comment:
there were some issues with getting those things centered. I'll revisit it and see if I can figure out the root cause.